torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -8902,7 +8902,7 @@ class MyDataClass:
             class DCModule(torch.nn.Module):
                 def __init__(self):
                     super().__init__()
-                    self.m = torch.nn.Linear(10, 10)
+                    self.m = torch.nn.Linear(10, 10, bias=False)
```

Comment:
this change is mainly to test that the autograd cb indeed gets enqueued if there is only one param that gets grad, as expected.